### PR TITLE
Prepare karate for tests with multiple backends running at the same time

### PR DESCRIPTION
**Pull request by 4xiom5**
- Modified the karate Server classes to allow multiple server running at the same time.
- Updated both backends to add some missing configuration options

### CODE REVIEW

File: be1-go/cli/cli.go [hunk 1]
```diff
@@ -69,10 +71,12 @@ func (s *ServerConfig) newHub(l *zerolog.Logger) (hub.Hub, error) {
...
-
-	if s.ClientPort == 9002 {
-		path = "./database-b/" + sqlite.DefaultPath
+	if s.DatabasePath == "" {
+		if s.ClientPort == 9002 {
+			s.DatabasePath = "./database-b/" + sqlite.DefaultPath
+		} else {
+			s.DatabasePath = "./database-a/" + sqlite.DefaultPath
+		}
 	}
```
Comment (emonnin-epfl) [inline be1-go/cli/cli.go:80]: Could you add the datapath to both configFile1 and configFile2 and then remove the corresponding lines?

**Review by onsriahi14 — COMMENTED** [review]
Based on my understanding, the code appeared to be good to me.

**Review by emonnin-epfl — APPROVED** [review]

File: be1-go/cli/cli.go (hunk 1 above)
Comment (4xiom5) [reply be1-go/cli/cli.go]: Done

**Review by simone-kalbermatter — APPROVED** [review]
LGTM!

**Pull request merged**